### DISCUSSION TITLE
Unable to configure two same REST endpoints with different http method [GET,POST]

### DISCUSSION THREAD

**Discussion by sunilg009** [microcks/microcks#913] [Q&A]
I have added 2 REST Endpoints with same name , where having different http method GET,POST. 
ex:= 
API1 [GET] = http://localhost:8080/logData 
API2 [POST] = http://localhost:8080/logData  [with Request Body]

Issue facing in after uploading the xml in microcks cluster.
- we see that one request endpoint [GET] is exposed , Instead of 2 request endpoints.

**Comment by arunmaersk** [comment]
facing the same issue, can someone provide any solution or workaround....

**Reply by lbroudoux** [reply]
An issue has been opened by @sunilg009 here: https://github.com/microcks/microcks/issues/914
Please provide complimentary information so that we can try to reproduce it.
